gobre/recipe/template/__init__.py

```diff
@@ -90,12 +89,16 @@ def strip_dict(d):
         target_files = split(target_file_option)
         if len(template_files) != len(target_files):
             raise zc.buildout.UserError(
-                "The number of template and target files must match"
+                "The number of template and target files must match",
```

Comment:
Is it possible to disable this style rule? IMHO it is one of the silliest rules to have a trailing comma in function calls as this is totally redundant and we have good enough diff tools now so there is no need to keep this old rule of possibly reducing future diffs.